packages/zowe-explorer/i18n/sample/src/globals.i18n.json

```diff
@@ -15,5 +15,8 @@
   "createFile.attribute.secondary": "Enter the secondary space allocation",
   "createFile.attribute.size": "Enter the size of the data set",
   "createFile.attribute.storclass": "Enter the SMS storage class",
-  "createFile.attribute.volser": "Enter the volume serial on which the data set should be placed"
+  "createFile.attribute.volser": "Enter the volume serial on which the data set should be placed",
+  "searchJobs.owner.id": "Enter job owner id",
```

Comment:
Minor suggestion: might want to standardize capitalization of ID to keep consistency across help/error messages

Reply:
I can change them all to lower case. 
At IBM we follow writing guidelines that say to only capitalize words in very specific cases, generic reference to an id would not need capitalization: https://carbondesignsystem.com/guidelines/content/writing-style/#capitalization